ISSUER_PROTOCOL.md

```diff
@@ -479,6 +486,6 @@ struct {
 
 ## Version History
 
-V3 uses [`ecdsa_secp256r1_sha256`](https://tools.ietf.org/html/rfc8446#section-4.2.3) as the signing algorithm for request signing.
+V3 uses [`ecdsa_secp256r1_sha256`](https://tools.ietf.org/html/rfc8446#section-4.2.3) as the signing algorithm for request signing and updates the key commitment format.
```

Comment:
Can we add a little more detail about how we update the key commitment format?

Reply:
Updated.